contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -35,25 +39,40 @@ def register_options(cls, register):
       register('--version', advanced=True, default='6.9.1',
                help='Node distribution version.  Used as part of the path to lookup the '
                     'distribution with --binary-util-baseurls and --pants-bootstrapdir')
+      register('--package-manager', advanced=True, default='npm',
+               help='Default package manager config for repo. Should be one of [npm,yarnpkg]')
```

Comment:
`register(..)` supports a `choices` arg: `choices=['npm', 'yarnpkg']`.

Also, you should probably pass `fingerprint=True` for all of these options.